benchmark/Utils.php

```diff
@@ -4,41 +4,43 @@
 
 use MongoDB\Client;
 use MongoDB\Collection;
+use MongoDB\Database;
 
 use function getenv;
 
 final class Utils
 {
     private static ?Client $client;
+    private static ?Database $database;
     private static ?Collection $collection;
 
     public static function getClient(): Client
     {
-        return self::$client ??= self::createClient();
+        return self::$client ??= new Client(self::getUri());
+    }
+
+    public static function getDatabase(): Database
```

Comment:
Refactored to give access to an instance of each library objects, and cache them.